style.css

```diff
@@ -0,0 +1,394 @@
+body {
+  display: flex;
+  flex-direction: column;
+  width: 390px;
+  height: auto;
+  padding: 0;
+  margin: 0;
+  background-color: rgb(245, 245, 245);
+  overflow-y: scroll;
+  scroll-behavior: smooth;
+}
```

Comment:
- [ ] Great job 👍 creating a portfolio. Unfortunately, it doesn't follow any of the templates provided by Microverse. Please go through all templates and choose one which you would follow to design your portfolio. 🙏  [Microverse templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)